Framework/Algorithms/test/ReflectometryReductionOne2Test.h

```diff
@@ -556,14 +556,14 @@ class ReflectometryReductionOne2Test : public CxxTest::TestSuite {
     alg.setProperty("CorrectionAlgorithm", "ExponentialCorrection");
     alg.setProperty("C0", 0.2);
     alg.setProperty("C1", 0.1);
-    MatrixWorkspace_sptr outLam = runAlgorithmLam(alg, 11);
+    MatrixWorkspace_sptr outLam = runAlgorithmLam(alg, 8);
 
-    TS_ASSERT_DELTA(outLam->x(0)[0], 0.920496, 1e-6);
-    TS_ASSERT_DELTA(outLam->x(0)[3], 5.159104, 1e-6);
-    TS_ASSERT_DELTA(outLam->x(0)[7], 10.810581, 1e-6);
-    TS_ASSERT_DELTA(outLam->y(0)[0], 16.351599, 1e-6);
-    TS_ASSERT_DELTA(outLam->y(0)[3], 23.963539, 1e-6);
-    TS_ASSERT_DELTA(outLam->y(0)[7], 39.756738, 1e-6);
+    TS_ASSERT_DELTA(outLam->x(0)[0], 2.333365, 1e-6);
```

Comment:
If it's possible to derive these values rather than hard-code them I would do that as makes tests more readable. See http://docs.mantidproject.org/nightly/concepts/UnitFactory.html